### PR TITLE
Add a db tag guidance.

### PR DESCRIPTION
**Pull request by wu-sheng**
Right now, include two demonstrations.
- Java, `Spring JdbcTemplate -> MySQL`
- Python, `redis-py  -> Redis`

### CODE REVIEW

File: guidance/database/tags.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Database tags guidance
```
Comment (bhs) [inline guidance/database/tags.md]: I like the level of detail here, but I think we should wait to merge until the `data_conventions.yaml` becomes an .md file. I will try (?) to get that done today, assuming I can find the time!

Comment (wu-sheng) [inline guidance/database/tags.md]: I will wait for you. It's better, that you write the `data_conventions.md`.

Comment (bhs) [inline guidance/database/tags.md]: (https://github.com/opentracing/specification/pull/45)

**Comment by wu-sheng** [comment]
Closing it. Include in #45 .

**Pull request closed without merging**